ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml

```diff
@@ -32,6 +32,18 @@ tests:
     test:
     - ref: openshift-qe-cluster-density-v2
     workflow: rosa-aws-sts-hypershift
+- as: perfscale-rosa-multiaz-ingress-perf-3nodes
+  cron: 0 11 * * 5
```

Comment:
@krishvoor Any reason why we want to run these tests in prod? Sorry for any confusion, this test suite was added to specifically add any tests that need to run in prod as exception, otherwise, we always run in stage only as of now. Alternatively, we could leave the test here and remove cron, but set "always_run: false" and keep the test for adhoc testing. See https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml#L24 for example. 
I will update all 4.12 tests to avoid confusion.
FYI, this change will require you to rerun "make jobs".

Reply:
>we always run in stage only as of now.
Ack, updating the params, included ingress-job to reflect across prod/ candidate/ nightly releases

Reply:
Added.